### PR TITLE
Adding localized resources.

### PR DESCRIPTION
**Pull request by sivanatase1-ctxs**
adding localized resource files for 
MS_Dynamics
JIRA
Power_BI
ServiceNow
Zendesk
G_Suite_Directory
Salesforce_alpha

### CODE REVIEW

**Comment by sivanatase1-ctxs** [comment]
@mavocitrix @karthikeyan-g11n  @yiminl-ctx @yangwa-ctx please review the PR

**Review by karthikeyan-g11n — APPROVED** [review]
Reviewed the changes, the changes are fine and hence approving.

**Review by mavocitrix — APPROVED** [review]

File: bundles/http/Citrix/7da37e64-5472-41fc-b86d-8923dafe00e5/i18n/zh-CN.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "dfd179ed-bddd-4c9b-a269-2ebeb6a797e0": {
+    "3f0a059c-4aea-453f-9e09-e4120006fb10": "URL",
+    "5fff755e-31ea-4827-933a-e34d4541eb01": "Search Dashbard",
```
Comment (yangwa-ctx) [inline bundles/http/Citrix/7da37e64-5472-41fc-b86d-8923dafe00e5/i18n/zh-CN.json:4]: Why this string is not localized?

File: bundles/http/Citrix/aa190bbb-79e8-4281-8bc4-7d7941432e9c/i18n/zh-CN.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    "00895422-56d3-4a0c-8c93-cd0364f0ee59": "ID",
+    "c62606fa-04b6-479f-9f3c-e793b803b90f": "提交者",
+    "632c133e-89d0-4c23-b307-d09a48c70612": "说明",
+    "8cf3a7f9-424d-4c3d-bae9-af071aee18c3": "票证",
+    "aa85056b-1843-4127-9ca0-9cc82cf99938": "票证已更新",
+    "c760e7af-1d91-4c89-ab9f-e8e0ba7304ec": "票证 ID: {{id}} - {{subject}}",
+    "c526f82a-caae-4fe3-9c71-ab895c79af9e": "说明",
+    "9891eb45-5a84-4569-a278-eb90b0dfa899": "搜索票证",
+    "cd059fb2-7d9f-47fc-9963-caaae91f6cf5": "主题",
+    "0dd96f1f-d4d7-44ea-b396-dfcf8602b2d4": "Assigned to {{ticket_assignee_id_user.name}}",
```
Comment (yangwa-ctx) [inline bundles/http/Citrix/aa190bbb-79e8-4281-8bc4-7d7941432e9c/i18n/zh-CN.json:16]: Unlocalized

File: bundles/http/Citrix/c8afddd5-e247-4184-bc65-8c8a88b824c2/i18n/ja.json [hunk 3]
```diff
@@ -0,0 +1,145 @@
+{
+  "c31de16d-990b-400a-83ff-ddc5c53ef9bd": {
+    "7ab7eb5a-2606-4c72-995a-d0b77a068673": "Location",
```
Comment (yangwa-ctx) [inline bundles/http/Citrix/c8afddd5-e247-4184-bc65-8c8a88b824c2/i18n/ja.json:3]: Unlocalized.

**Review by yangwa-ctx — APPROVED** [review]
@sivanatase1, There are little strings that are not translated, please take a look.

**Review by ghost — APPROVED** [review]
The keys of other languages are the same as en, the mustache template has not been modified, and there are no illegal characters.

File: bundles/http/Citrix/7da37e64-5472-41fc-b86d-8923dafe00e5/i18n/zh-CN.json (hunk 1 above)
Comment (sivanatase1-ctxs) [reply bundles/http/Citrix/7da37e64-5472-41fc-b86d-8923dafe00e5/i18n/zh-CN.json:4]: this is a test PR. actual translations are in progress.

File: bundles/http/Citrix/aa190bbb-79e8-4281-8bc4-7d7941432e9c/i18n/zh-CN.json (hunk 2 above)
Comment (sivanatase1-ctxs) [reply bundles/http/Citrix/aa190bbb-79e8-4281-8bc4-7d7941432e9c/i18n/zh-CN.json:16]: @yangwa-ctx this is a test PR. actual translations are in progress.

File: bundles/http/Citrix/c8afddd5-e247-4184-bc65-8c8a88b824c2/i18n/ja.json (hunk 3 above)
Comment (sivanatase1-ctxs) [reply bundles/http/Citrix/c8afddd5-e247-4184-bc65-8c8a88b824c2/i18n/ja.json:3]: @yangwa-ctx this is a test PR. actual translations are in progress.

**Pull request merged**